be/src/storage/persistent_index.cpp

```diff
@@ -49,10 +49,19 @@ T pad(T v, P p) {
     return npad(v, p) * p;
 }
 
+uint32_t log2(size_t n) {
```

Comment:
util/bit_util.h have this implementation. You can use the util instead.

Reply:
> util/bit_util.h have this implementation. You can use the util instead.

done, change in https://github.com/StarRocks/starrocks/pull/4172/commits/19bb730532d645dd8192ac70522ad282327f364c